### PR TITLE
Register h3lios.is-a.dev

### PR DESCRIPTION
**Pull request by H3ilos**
Register h3lios.is-a.dev with CNAME record pointing to ninjaxility.github.io.

### CODE REVIEW

File: domains/h3lios.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "NinjaXility",
+           "email": "",
+           "discord": "511106356267319296",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.IBR3ZrvHG0dyQ1OU_ThEq2VbB_gvPhqPJkyPtwwbODCQENB-T2fstb7JrqYXG9lNaBWBrkeUzrheB5HY5tgEmPDAe6eDnN1D0dk0HKwxySEO7I1wbJsoJUnrWXigmuIYYkCPqHO9bBBD3seh4Y-REvCFI_xjNxBOIkqYiISDG4SJZW_AMoM-4ZwWJOS30ASQA3IM1DrvS3cNeamN0vmy2W16z7PUQxfn3UpwhdalfbzZKgiPowTVnbMT8OZ0IL45VZCmqeikraFf7fUv0PPCk54c07A8mIpYepzyy4hUftsF-ILHDLU_iOMZDHPrMDpPeyFXgYjA1iv1AfgFj6lshA.CFjZ-C3qr_js2SqYJQruNQ.j12mMFCkJhwZpCJXE_JDaY1XgeCUkSwhowfAvMEFrxvTdeqGbeEaHiXN4-oXfM0FuPLNgfsJDTwmva-ekWTAeOtX4a3IBlJ0uuj65_tdMOw.NDTU5ISRijrUIUoDvdYXJw"
+        },
+    
+        "record": {
+            "CNAME": "https://ninjaxility.github.io/"
```
Comment (DEV-DIBSTER) [inline domains/h3lios.json:10]: CNAME must be a hostname with no protocols or file paths. See docs: https://docs.is-a.dev

**Review by DEV-DIBSTER — DISMISSED** [review]

**Review by H3ilos — COMMENTED** [review]
fixed it :)

File: domains/h3lios.json (hunk 1 above)
Comment (orxngc) [reply domains/h3lios.json:10]: What Dibster said here. And no, you didn't fix it.

**Comment by NinjaXility** [comment]
should work now

File: domains/h3lios.json (hunk 1 above)
Comment (CuteDog5695) [inline domains/h3lios.json:10]: This is what you needed to do
```suggestion
            "CNAME": "ninjaxility.github.io"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/h3lios.json (hunk 1 above)
Comment (NinjaXility) [reply domains/h3lios.json]: fixed in my fork 
image

**Review by iostpa — CHANGES_REQUESTED** [review]
Please add more stuff to your website.

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**